src/main/java/uk/gov/ch/exception/TrustDataCountNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ch.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class TrustDataCountNotFoundException extends Exception {
```

Comment:
TrustDataNotFoundException

Reply:
The other exceptions in this repo follow a similar pattern for the not found exceptions so i was just following it here

Reply:
I did not see any *Count*Exceptions but if so fine.